packages/telemetry/src/telemetryReporter.ts

```diff
@@ -128,12 +99,37 @@ export class TelemetryReporter extends AsyncCreatable<TelemetryOptions> {
    * @param properties {Properties} - map of properties to publish alongside the event.
    */
   public sendTelemetryMetric(metricName: string, value: number, properties?: Properties): void {
+    this.sendTelemetry(TelemetryMethod.METRIC, metricName, { name: metricName, value, properties });
+  }
+
+  private sendTelemetry(
+    method: TelemetryMethod,
+    message: string,
+    data: EventTelemetry | ExceptionTelemetry | MetricTelemetry | TraceTelemetry
```

Comment:
You can alias this into its own type, e.g.

```typescript
type TelemetryData = EventTelemetry | ExceptionTelemetry | MetricTelemetry | TraceTelemetry;
```

Reply:
@mdonnalley Oooh thanks good idea. See `fix: alias telemetry types as TelemetryData`